platform/commonUI/browse/test/InspectorPaneControllerSpec.js

```diff
@@ -71,7 +73,12 @@ define(
                 mockDomainObject.hasCapability.andReturn(true);
                 mockDomainObject.getCapability.andReturn(mockStatusCapability);
 
-                controller = new InspectorPaneController(mockScope, mockAgentService, mockWindow, mockNavigationService);
+                mockLocation = jasmine.createSpyObj("$location", ["search"]);
+                mockLocation.search.andReturn({});
+
+                mockAttrs = jasmine.createSpyObj("$attrs", ["alias"]);
```

Comment:
a jasmine spy object is an object with spy properties: actual functions.  You don't need that for your tests, you can just use a regular object, i.e. `mockAttrs = {};`